bin/man-n

```diff
@@ -4,6 +4,8 @@
 # Constants.
 #
 
+readonly location="$_" # this must be the first command executed
```

Comment:
How does this work? `$_` returns only the last token of the previous argument, e.g.:

`echo foo bar baz`
> foo bar baz
`echo $_`
> baz